<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Currencies</class>
 <widget class="QWidget" name="Currencies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RANKING</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableRanking">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>691</width>
     <height>421</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="startButtonCurrencies">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>500</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#34495e;color:white;margin:0px;text-align:center;padding:2px;border-radius:3px;width:100px;font-size:11px;font-weight:600;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/res/icons/update.png</normaloff>:/icons/res/icons/update.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblLoading">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>500</y>
     <width>46</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
